pkg/vault/test/foundry/fuzz/AddAndRemoveLiquidity.medusa.sol

```diff
@@ -0,0 +1,382 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import "forge-std/Test.sol";
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import { IBasePool } from "@balancer-labs/v3-interfaces/contracts/vault/IBasePool.sol";
+import { Rounding } from "@balancer-labs/v3-interfaces/contracts/vault/VaultTypes.sol";
+
+import { FixedPoint } from "@balancer-labs/v3-solidity-utils/contracts/math/FixedPoint.sol";
+import { InputHelpers } from "@balancer-labs/v3-solidity-utils/contracts/helpers/InputHelpers.sol";
+
+import { BasePoolMath } from "../../../contracts/BasePoolMath.sol";
+import { BalancerPoolToken } from "../../../contracts/BalancerPoolToken.sol";
+
+import "../utils/BaseMedusaTest.sol";
+
+contract AddAndRemoveLiquidityMedusaTest is BaseMedusaTest {
+    using FixedPoint for uint256;
+
+    uint256 private constant _MAX_BALANCE = 2 ** (128) - 1; // from PackedTokenBalance.sol
```

Comment:
Yes; no need to redefine a constant we already have. Maybe just use MAX_UINT128 directly, rather than assigning constants to each other (perhaps with a comment on what it means)

Reply:
So, I can't use the MAX_UINT128 constant, because that's on BaseTest of forge, which is not used by the Medusa tests. But, I replaced by type(uint128).max